### PR TITLE
rename foot_offset to foot_shift

### PR DESCRIPTION
**Pull request by Louka3000**
For https://github.com/SlimeVR/SlimeVR-Server/pull/199

### CODE REVIEW

File: protocol/flatbuffers/rpc.fbs [hunk 1]
```diff
@@ -114,7 +114,7 @@ enum SkeletonBone: uint8 {
     LEGS_LENGTH = 8,
     KNEE_HEIGHT = 9,
     FOOT_LENGTH = 10,
-    FOOT_OFFSET = 11,
+    FOOT_SHIFT = 11,
```
Comment (TheButlah) [inline protocol/flatbuffers/rpc.fbs:117]: IMO shift is not clear, offset is.

**Comment by TheButlah** [comment]
Punting thorough review it seems like there are non-bone related things in here already. This needs to be looked at again later.

**Review by TheButlah — APPROVED** [review]
Punting because it seems like there are non-bone related things in here already. This needs to be looked at again later.

File: protocol/flatbuffers/rpc.fbs (hunk 1 above)
Comment (Louka3000) [reply protocol/flatbuffers/rpc.fbs:117]: FOOT_SHIFT shifts the foot's head node backwards or forward, affecting the whole foot and not just its computed tracker.

OFFSETs offset tracker nodes only, without affecting the rest of the skeleton.

**Pull request merged**